packages/web-components/fast-foundation/src/data-grid/data-grid.ts

```diff
@@ -126,6 +126,32 @@ export class DataGrid extends FoundationElement {
         return templateColumns;
     }
 
+    /**
+     * When true the component will not add itself to the tab queue.
+     * Default is false.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: no-tabbing
+     */
+    @attr({ attribute: "no-tabbing", mode: "boolean" })
+    public noTabbing: boolean = false;
+    private noTabbingChanged(): void {
+        if (this.$fastController.isConnected) {
+            if (this.noTabbing) {
+                this.setAttribute("tabIndex", "-1");
```

Comment:
Should this just not set tabindex? This would allow the developer to still control the tabindex if they wanted to. They might want to elevate a particular cell in the tabbing order? Not sure of use cases, just wondering if we can make it more flexible if the case is needed.

Reply:
I think we want to keep the data-grid as a focusable entity whether or not it is in the tab queue.